proposals/2024-07-15-otel-resource-attribute-promotion.md

```diff
@@ -0,0 +1,89 @@
+# OTel resource attribute promotion
+
+* **Owners:**
+  * Arve Knudsen [@aknuds1](https://github.com/aknuds1) [arve.knudsen@grafana.com](mailto:arve.knudsen@grafana.com)
+
+* **Implementation Status:** Partially implemented
+
+* **Related Issues and PRs:**
+  * [WIP: OTLP Translator prometheusremotewrite: Support resource attribute promotion](https://github.com/prometheus/prometheus/pull/14200)
+
+* **Other docs or links:**
+
+> This proposal collects the requirements and implementation proposals for supporting OTel resource attribute promotion to labels.
+
+## Why
+
+Currently, Prometheus encodes OpenTelemetry (OTel for short) [resource attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/sdk.md) as labels of the `target_info` metric.
+OTel resource attributes model metadata about the environment producing metrics received by the backend (e.g. Prometheus).
```

Comment:
Can we add that resource attributes are akin to target_labels coming from Service Discovery mechanism, and we copy them over even in the Pull method? So doing it via Push isn't that different.

Reply:
@gouthamve could you provide me with a suggestion for what to add, please? I don't know much about service discovery :)

Reply:
Done.